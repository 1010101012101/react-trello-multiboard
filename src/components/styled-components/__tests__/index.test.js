import React from 'react'
import { shallow } from 'enzyme'
// https://github.com/styled-components/jest-styled-components
import 'jest-styled-components'

import { AppContainer, BlockContainer, ColumnContainer, Container, ListContainer } from '../index'

describe('StyledComponents', () => {
  test('should render an AppContainer with the proper styling', () => {
    expect(shallow(<AppContainer />)).toMatchSnapshot()
  })

  test('should render an BlockContainer with the proper styling', () => {
    expect(shallow(<BlockContainer />)).toMatchSnapshot()
  })

  test('should render an ColumnContainer with the proper styling', () => {
    expect(shallow(<ColumnContainer />)).toMatchSnapshot()
  })

  test('should render an Container with the proper styling', () => {
    expect(shallow(<Container />)).toMatchSnapshot()
  })

  test('should render an ListContainer with the proper styling', () => {
    expect(shallow(<ListContainer />)).toMatchSnapshot()
  })
})
