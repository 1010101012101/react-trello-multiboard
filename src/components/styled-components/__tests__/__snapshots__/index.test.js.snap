// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledComponents should render an AppContainer with the proper styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;

exports[`StyledComponents should render an BlockContainer with the proper styling 1`] = `
.c0 {
  display: block;
  padding: 10px 10px;
}

<div
  className="c0"
/>
`;

exports[`StyledComponents should render an ColumnContainer with the proper styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;

exports[`StyledComponents should render an Container with the proper styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  overflow-x: auto;
  padding: 10px 10px;
}

<div
  className="c0"
/>
`;

exports[`StyledComponents should render an ListContainer with the proper styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  overflow-x: auto;
  padding: 10px 10px;
}

@media (max-width:768px) {
  .c0 {
    margin: 0 -10px;
  }
}

<div
  className="c0"
/>
`;
